src/plugins/cordova-plugin-device-orientation/sim-host.js

```diff
@@ -32,7 +32,7 @@ module.exports = function (messages) {
         });
         compassWidget.initialize(compass.heading);
 
-        inputHeading.addEventListener('input', function () {
+        inputHeading.addEventListener('change', function () {
```

Comment:
It would be highly desirable to add these changes in a separate commits. As you spot these, you could `git add -p` and `git commit` them in situ so they don't mix with the changes of other nature. It would make bisecting later so much easier and the commits more atomic. Then we don't have to go back to PR comments to see why they appeared in the code in a middle of something that does not seem quite relevant.